pkg/engine/ahocorasick/ahocorasickcore.go

```diff
@@ -1,9 +1,10 @@
-package engine
+package ahocorasick
```

Comment:
This had to be moved out of engine, otherwise it was not possible to add keyword detection to the detector tests.